net/wireguard-tools/Portfile

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                wireguard-tools
+# please only update version when the tools have been modified/updated,
+# since the Linux module aspect isn't of utility for macOS
```

Comment:
I cannot understand this comment, which probably means other people won't be able to either. Please rephrase this. Remember that your comments like this are likely to be read by people who know less than you do about a port.